docs/develop/integrations/cloudflare.md

```diff
@@ -0,0 +1,216 @@
+---
+sidebar_position: 7
+sidebar_label: Cloudflare
+title: Integrating Momento with your Cloudflare worker
+description: Learn to deploy a Cloudflare worker that uses Momento HTTP API or Web SDK
+pagination_next: null
+---
+
+[Cloudflare Workers](https://workers.cloudflare.com/) is a popular platform for deploying and running serverless code.
+
+In this tutorial, we'll make use of two simple [examples](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/cloudflare-workers) from our [Javascript SDK](https://github.com/momentohq/client-sdk-javascript). By the end of the exercise, you'll have a Cloudflare worker that interacts with Momento cache to populate and retrieve data.
+
+## Prerequisites
+
+To get this app deployed and running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A [Cloudflare](https://workers.cloudflare.com/) account.
+- A copy or fork of the [Momento Javascript SDK](https://github.com/momentohq/client-sdk-javascript) in your personal repositories.
+
+Once you have a Cloudflare account and a copy or fork of the Momento Javascript SDK in your Git provider account, you're ready to configure the Momento side of things using the [Momento console](https://console.gomomento.com). You can create an account in the console by providing your email address or linking an existing Google or GitHub account. Once you've logged into the console, click the `Create Cache` button on the top right of the page:
+
+![Create Cache button](/img/console-create-cache.png)
+
+Create a cache called `worker`. You can create the cache in your choice of cloud provider, AWS or GCP, and in any region.
+
+![Create cache form](/img/console-create-worker-cache-form.png)
+
+After pressing the `Create` button you'll see the new `worker` cache in the list of available caches.
+
+![Chat cache](/img/console-caches-worker.png)
+
+Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache, choose the `Super User Token` token type, and hit the `Generate Token` button.
```

Comment:
Why a super-user token and not a narrowly scoped FGAC token? A super-user token sounds like it might give too much access.

Reply:
Discussed on slack to fast follow for this and Vercel/Netlify ones